lte/gateway/c/core/oai/include/spgw_types.hpp

```diff
@@ -82,9 +84,8 @@ typedef struct sgw_s11_teid_s {
 } sgw_s11_teid_t;
 
 // Map- Key:teid (uint32_t) ,
-// Data:s_plus_p_gw_eps_bearer_context_information_s*
-typedef magma::proto_map_s<uint32_t,
-                           struct s_plus_p_gw_eps_bearer_context_information_s*>
+// Data: S11BearerContext*
+typedef magma::proto_map_s<uint32_t, magma::lte::oai::S11BearerContext*>
```

Comment:
I would prefer to keep the existing name because it's been there in older builds also. With changing name we need deprecate the existing field. The comment in proto description explains it includes both sgw's and pgw's contexts. 
More over it's not bearer context, it's the session context. So we need to change all fields. If it's really required we can take up in separate PR

Reply:
In the older builds the name is s_plus_p_gw_eps_bearer_context_information right? I think the S11 prefix should be atleast changed as it is not related to s11 interface.

Reply:
Yes it's not related S11 interface and also it's not bearer context. If we need to change let's change it completely including SgwEpsBearerContextInfo and PgwEpsBearerContextInfo. That can be taken up in separate PR. Let's not club many changes in single PR

Reply:
Sure. Please add a TODO to take it up in a new PR